setup.cfg

```diff
@@ -1,5 +1,66 @@
+[metadata]
+name = pyroma
+version = 3.3.dev0
+description = Test your project's packaging friendliness
+long_description = file: README.rst, CHANGES.txt
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: Implementation :: PyPy
+keywords = pypi, quality, testing
+author = Lennart Regebro
+author_email = regebro@gmail.com
+url = https://github.com/regebro/pyroma
+license = MIT
+project_urls =
+    Source Code = https://github.com/regebro/pyroma
+
+[options]
+zip_safe = True
+include_package_data = True
+packages = find:
+package_dir =
+    = .
+test_suite = pyroma.tests
+install_requires =
+    setuptools
+    docutils
+    pygments
+    requests
+
+[options.packages.find]
+where = .
+
+[options.extras_require]
+test =
+    pytest
+    pytest-cov
```

Comment:
Hey @regebro , was this added by mistake? As far as I can tell, Pyroma doesn't actually require Pytest; it doesn't use it in its tests, and the CIs run the tests just fine with vanilla unittest.

Reply:
That sounds like a cut and paste error on my part, yes.

Reply:
Thanks, fixed in PR #92